Documentation/media/v4l-drivers/bcm2835-isp.rst

```diff
@@ -0,0 +1,118 @@
+.. SPDX-License-Identifier: GPL-2.0
+
+BCM2835 ISP Driver
+==================
+
+Introduction
+------------
+
+The BCM2835 Image Sensor Pipeline (ISP) is a fixed function hardware pipeline
+for performing image processing operations. Input frames (in Bayer format) may
+be obtained from the BCM2835 Unicam peripheral, written to a memory framebuffer,
+and fed into the input of the ISP.  Two output images may be generated (in
+YUV or RGB format), along with image statistics.
+
+The bcm2835-isp driver exposes the following media pads as V4L2 device nodes:
+
+.. tabularcolumns:: |l|l|l|l|
+
+.. cssclass: longtable
+
+.. flat-table::
+
+    * - *Pad*
+      - *Direction*
+      - *Purpose*
+      - *Formats*
+
+    * - "bcm2835-isp0-output0"
+      - sink
+      - Accepts Bayer format frame buffers as input to the ISP HW pipeline.
+      - :ref:`RAW8 <V4L2-PIX-FMT-SRGGB8>`,
+        :ref:`RAW10 <V4L2-PIX-FMT-SRGGB10>`,
+        :ref:`RAW10P <V4L2-PIX-FMT-SRGGB10P>`,
+        :ref:`RAW12 <V4L2-PIX-FMT-SRGGB12>`,
+        :ref:`RAW12P <V4L2-PIX-FMT-SRGGB12P>`,
+        :ref:`RAW14 <V4L2-PIX-FMT-SRGGB14P>`,
+        :ref:`RAW16 <V4L2-PIX-FMT-SRGGB16>`.
```

Comment:
You are right we don't support it currently.  I will remove.

If we do need support, it is trivial.  The ISP HW will handle everything automatically (given the correct input format is selected) since Unicam will not shift the pixels up, only pack into 16-bits.